test/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:12
+
+ENV password=notpassword1
+ENV dbpass=notpassword2
+ENV pswd=notpassword3
+ENV pass=notpassword4
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High <!-- bcec2d5d-92d3-41cf-b2f1-a6c961c70a10 -->